bootstrap/src/main/scala/org/scalafmt/bootstrap/Bootstrap.scala

```diff
@@ -77,7 +77,7 @@ object ScalafmtBootstrap {
         logger = Some(logger)
       )
     )
-    val resolution = start.process.run(fetch).unsafePerformSync
```

Comment:
No, that doesn't guarantee that different threads see the same values for `cliCache`.